docs/guides/api/api.primary.in.rst

```diff
@@ -0,0 +1,51 @@
+
+**Use the API Aspect to:**
+
+- Access Rubin data from outside the RSP.
+- Automate data queries or downloads.
+- Integrate Rubin data access into your own software tools.
+- Access external datasets in addition to Rubin data.
+
+API Services
+************
+
+The API aspect of the RSP supports several types of services, primarily based on the `International Virtual Observatory Alliance <https://www.ivoa.net/>`_ (IVOA) protocols.
+These include the `Table Access Protocol <https://www.ivoa.net/documents/TAP/20190927/index.html>`_ (TAP),  the `Observation Table Access Protocol <https://www.ivoa.net/documents/ObsCore/>`_ (ObsTAP), the `Server-side Operations for Data Access <https://www.ivoa.net/documents/SODA/20170517/index.html>`_ (SODA) service (which provides image cutouts and mosaics), the `Simple Image Access Protocol version 2 <https://www.ivoa.net/documents/SIA/20150730/index.html>`_ (SIAv2) for image searches, and the `Hierarchical Progressive Survey <https://www.ivoa.net/documents/HiPS/>`_ (HiPS).
+
+By the time of the first data release, the Rubin Science Platform will also support the `Simple Cone Search (SCS) <https://www.ivoa.net/documents/latest/ConeSearch.html>`_ for basic catalog queries, and the `Virtual Observatory Space <https://www.ivoa.net/documents/VOSpace/>`_ (VOSpace)—alongside `Web Distributed Authoring and Versioning <https://en.wikipedia.org/wiki/WebDAV>`_ (WebDAV)—for user file access.
+
+
+Table Access Protocol (TAP) Service
+===================================
+
+The `Table Access Protocol <https://www.ivoa.net/documents/TAP/20190927/index.html>`_ (TAP) is a specific type of service within the API, implemented according to the `IVOA <https://www.ivoa.net/>`_ standard.
+
+Internal TAP Access
+===================
+
+The primary means of accessing TAP within the RSP is through the RSP’s Portal and Notebook aspects.
+In the Notebook aspect, a client that's connected to the TAP service is instantiated within a Python notebook and used to execute an `Astronomical Data Query Language <https://www.ivoa.net/documents/ADQL/>`_ (ADQL) query, returning a result set.
+
+External TAP Access
+===================
+
+The RSP TAP service is also accessible remotely, from outside the RSP (e.g., from a personal computer), allowing users to query data using external software such as `PyVO <https://pyvo.readthedocs.io/en/latest>`_, `TOPCAT <https://www.star.bris.ac.uk/~mbt/topcat/>`_, or custom scripts.
+To access TAP from external software, users must generate an RSP access token.
+
+Authentication and Access Tokens
+--------------------------------
+
+Access to Rubin data via the API requires authentication.
+Users must obtain an RSP authentication token, which serves as a credential when making API requests.
```

Comment:
I would add a sentence saying that the token for Internal TAP access is already generated by the RSP so no additional steps are needed, either here or in the Internal TAP Access section above.

Reply:
Thanks for the suggestion. The text in this section now reads: 

> Access to Rubin data via the API requires authentication. Users must obtain an RSP authentication token, which serves as a credential when making API requests. For internal TAP access, the token is already generated by the RSP, so no additional steps are needed. For external TAP access, instructions on generating and using an RSP access token can be found on the [Authentication website](https://rsp.lsst.io/guides/auth/index.html).